.isort.cfg

```diff
@@ -0,0 +1,15 @@
+[settings]
+
+# Play nice with Black
+profile = black
+
+# Put two black lines after imports
+lines_after_imports = 2
+
+# Match max-line-length in Flake8 config
+line_length = 80
+
+# Default puts 1st party before 3rd party, this reverses that and adds
+# Twisted as a second party in the middle.
+known_secondparty = twisted
+sections = FUTURE,STDLIB,THIRDPARTY,SECONDPARTY,FIRSTPARTY,LOCALFOLDER
```

Comment:
Does klein import the reactor it would be good to have that on a separate grouping

Reply:
One-line-per-import felt like more change than I wanted to make.

Not sure if it's possible to do that only for reactor imports, but you could change the config to import-per-line, so your searching and make changes, then change the config back, and have just your relevant diff remaining… probably.  :-)